client/pages/torrent/[infoHash].js

```diff
@@ -523,7 +523,10 @@ const Torrent = ({ token, torrent = {}, userId, userRole, uid, userStats }) => {
       userStats.ratio < Number(SQ_MINIMUM_RATIO)) ||
     (Number(SQ_MAXIMUM_HIT_N_RUNS) !== -1 &&
       userStats.hitnruns > Number(SQ_MAXIMUM_HIT_N_RUNS));
-
+  function isPngImage(data) {
```

Comment:
Please follow existing code style here and include blank lines before & after function